bonus/bonus.js

```diff
@@ -3,15 +3,31 @@
  * @param {string}
  * @returns {string}
  */
-function replaceVowels() {}
+function replaceVowels(string) {
+  let newStr = "";
+  for (let i = 0; i < string.length; i++) {
+    if ("aeiou".includes(string[i].toLowerCase())) {
+      newStr += string[i].replace(/[aeiou]/gi, "*");
```

Comment:
This is good, but make sure you understand how this works and could replicate it in an interview if they asked this question and you couldn't use the internet. That should always be our goal.